en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
 # Create Your First Service
 
-To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.
+To learn how to create a service, let's look at a simple scenario. In this example, you'll create a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
 
 ## Step 1: Create the service
 
-To create the service, follow the procedure below: 
+To create the service, follow this procedure: 
 
-1. Access the Choreo Console via https://console.choreo.dev/.
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
     
-2. Sign in using either your Google or GitHub credentials.
-    
-3. In the **Services** page, click **Create**. You are directed to the low-code view.
+2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
 
-4. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
 
-5. Select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
+4. Select **GET** as the HTTP method and enter `activecases` as the relative path from the host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
-    
-    Then click **Save API**. Now you have saved the API trigger that starts your service.
-    
-6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+     ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+
+5. Click **Save API**. Now you have saved the API trigger that starts your service.
+
+6. Click **API Calls** and then complete the following steps to configure the API trigger to use the HTTP connection that connects the public API used in this scenario to it:
```

Comment:
Should we make this a sub section? Configure the service maybe?

Reply:
I feel the sentence is too long. Can we re-phrase it if we are not going to make it a sub section?

Reply:
+1 to have a subsection. But need to go through the complete flow and see the best way to do this.

Reply:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108